wsapi.go

```diff
@@ -426,12 +428,22 @@ type requestGuildMembersOp struct {
 // query     : String that username starts with, leave empty to return all members
 // limit     : Max number of items to return, or 0 to request all members matched
 // presences : Whether to request presences of guild members
-func (s *Session) RequestGuildMembers(guildID string, query string, limit int, presences bool) (err error) {
+// userIDs   : Used to specify which users you wish to fetch
+// nonce     : Nonce to identify the Guild Members Chunk response
+func (s *Session) RequestGuildMembers(guildID, query string, limit int, presences bool, userIDs *[]string, nonce string) (err error) {
+	// One of query or user_ids
+	var queryPt *string
+	if userIDs == nil {
+		queryPt = &query
+	}
```

Comment:
`omitempty` will not include `query` if the string is empty, hence the usage of pointer is to prevent the field being omitted when it is explicitly set to `""` (return all members). 

The endpoint will throw an error if both query and userIDs are not provided.